### PR TITLE
Event processing document re-write

### PR DESCRIPTION
**Pull request by vogella**
Event_Processing used to be a working document for the design of the Event Admin usasge in the Eclipse platform. As this document is linked by lots of other document it has been updated to reflect the decisions taken in the past, rather than posting these questions as something still to be solved.

### CODE REVIEW

File: docs/Event_Processing.md [hunk 1]
```diff
@@ -43,59 +42,44 @@ The main difference is that an intermediary is introduced between the sender and
...
+OSGi's Event Admin was adopted. 
+It is relatively simple, well described by the OSGi spec, with the reference implementation already in the Eclipse repository.
 
-Consider this specific problem in conforming to the spec: in the Event Admin, every single event needs to have a Map added to it. And it can't be null or an empty Map either, as the spec says that it includes the event's topic name.
+According to the the Event Admin, every single event needs to have a Map added to it. 
+And it can't be null or an empty Map either, as the spec says that it includes the event's topic name.
 
-After some experimenting I found that actual memory churn added by the EventAdmin is not too bad. In absolute numbers, its dispatch times were in the range of about 10 micro seconds per event for the expected load. Garbage collection was fine on long runs, probably adding an extra minor GC every 15 seconds, with the total GC time not affected in a measurable way.
+The dispatch times in EventAdmin  in absolute numbers were in the range of about 10 micro seconds per event for the expected load.
```
Comment (BeckerWdf) [inline docs/Event_Processing.md]:
```suggestion
The dispatch times in EventAdmin in absolute numbers were in the range of about 10 micro seconds per event for the expected load. 
```

**Review by BeckerWdf — CHANGES_REQUESTED** [review]
minor nitpic

File: docs/Event_Processing.md (hunk 1 above)
Comment (vogella) [reply docs/Event_Processing.md]: Thanks. Applied

**Comment by vogella** [comment]
@BeckerWdf applied, please release your blocking vote

**Review by BeckerWdf — APPROVED** [review]

**Pull request merged**